astropy/io/votable/tree.py

```diff
@@ -3268,10 +3412,10 @@ def version(self):
     @version.setter
     def version(self, version):
         version = str(version)
-        if version not in ('1.1', '1.2', '1.3'):
+        if version not in ('1.1', '1.2', '1.3', '1.4'):
             raise ValueError(
                 "astropy.io.votable only supports VOTable versions "
-                "1.1, 1.2 and 1.3")
+                "1.1, 1.2, 1.3 and 1.4")
```

Comment:
```suggestion
                "1.1, 1.2, 1.3, and 1.4")
```

Reply:
Thanks for the review.  I've implemented all of the above suggestions.